app/views/cluster_infos/show.html.haml

```diff
@@ -0,0 +1,113 @@
+- filter = {is_done: true, is_error: {'$exists'=> false}}
+- fields = {fields: {result: 1}}
+- result = @experiment.simulation_runs.where(filter, fields).first
+- output_parameters=[]
+- if result.blank?
+  - output_parameters = ["No results found in completed simulation runs"]
+- else
+  - output_parameters = result.result.keys
+- input_parameters = @experiment.get_parameter_ids
+- simulation_array = params[:simulations].split(',').map{|s| s.to_i}.sort
+
+#clusterInfo.reveal-modal(data-reveal="")
+%a.close-reveal-modal &#215
+%h2
+  Details of cluster #{params[:cluster_id]}
+%table{:width => '100%', :style => 'border: 0px'}
```

Comment:
Nie trzeba zmieniać tego w tym pliku, ale na przyszłość: zamiast ``:width => '100%'``używajmy notacji: ``width: '100%'`` jeśli to możliwe.